typescript.js

```diff
@@ -1,32 +1,32 @@
-'use strict';
+"use strict";
```

Comment:
What happened with quotes here? We use single quotes, I think something went wrong with the configuration. (I should enable Prettier here, I guess.)

Reply:
:thinking: I think it pulled in a default prettier config in vscode maybe?

Reply:
Yeah, is your Prettier extension running on _all_ code? I ran into this with Harrison, he had `formatOnSave` in his personal VS Code settings, but it should only be enabled in repository-specific settings (`.vscode/settings.json`), so his Prettier extension was running everywhere.

Reply:
setting format-on-save globally is going to be a Common Thing, if repos depend on it being _off_, they should explicitly disable it IMO

Reply:
Loaded in prettier configs and ran it once, though don't have it setup to run always

And added the _disable_ of format-on-save to the local .vscode settings

Reply:
> if repos depend on it being off, they should explicitly disable it IMO
Prettier is an opt-in, not an opt-out, so in essence _every_ repository that doesn't explicitly have a Prettier configuration "depends on it being off," right? Maybe we can enforce that every 6RS repository must add a Prettier configuration, but we can't do anything about open source repositories, and it would be uncouth to submit a Prettier-formatted pull request to an open source project if they don't use Prettier, wouldn't it?

Aside from the philosophical aspect, how would we make a global `formatOnSave` setting work with our non-default Prettier configuration? I don't think there is a way to create a global Prettier configuration file in a user's environment. We might be able to use [the plugin's default settings](https://github.com/prettier/prettier-vscode/#configuration), but that might be awkward to scale (would `infrastructure` touch users' VS Code configurations?).

Reply:
My view is that the global format-on-save is a _cross-language_, user setting. Granted, such things can do damage, and users are, in the end, responsible for that.

But we can also make things easier for users. E.g. we also don't even know that users will have their default vscode formatter set to prettier in the first place :) But we _could_ include that setting in the repos where prettier is configured, and probably we should.

I don't think having `infrastructure` hack on peoples vscode settings automatically would be good, but having a tool there which _suggests_ the recommended settings and lets you yes/no to applying them would be useful, ditto suggesting recommended extensions to install :)

Reply:
Agreed, thanks for the thoughts. I had been thinking of it as a language-specific setting, but you're right that it's cross-language, good point!

Reply:
FWIW, there _are_ language-specific versions of that setting :)